ui/app/templates/components/license-banners.hbs

```diff
@@ -31,9 +31,9 @@
       }}"
       @marginless={{true}}
     >
-      <LearnLink @path="/tutorials/nomad/hashicorp-enterprise-license">
```

Comment:
Not sure why we were referencing Nomad docs here, was this intentional?

Reply:
I think vault didn't have a dedicated license page yet, so we used nomad's which has the same instructions for generating a license